### ISSUE TITLE
Ensure focusable elements are not hidden

### ISSUE THREAD

**Issue by schalkneethling** [mdn/mdn-fiori#85]
If an element can be focused to, ensure it does not have aria-hidden set to true. An example of this is the main MDN Web Docs logo.

## Bugzilla

https://bugzilla.mozilla.org/show_bug.cgi?id=1563469
